roads.mss

```diff
@@ -1394,152 +1397,310 @@ come in as well.
   }
 }
 
-#roads_high[zoom>=11],
+#roads_high::inline[zoom>=11],
 #tunnel::inline[zoom>=11],
 #bridge::inline[zoom>=11] {
-  /* -- colors & styles -- */
+  // surfaces on track and path
+  [type='track'],
+  [type='bridleway'],
+  [type='footway'],
+  [type='path'],
+  [type='cycleway'] {
+    [surface_type='unknown'][zoom >= 13],
+    [surface_type='cyclocross'][zoom >= 12],
+    [surface_type='mtb'][zoom >= 12] {
+      background/line-join: round;
+      background/line-opacity: 0.4;
```

Comment:
I'm missing something here, why is there an opacity for the background?

Reply:
Good question, with opacity the background can melt with the background map (landuse)

Reply:
check tracks between forest and land https://user-images.githubusercontent.com/47089717/80146366-0b640a80-85b2-11ea-9f86-7d695fba2355.png